src/App.tsx

```diff
@@ -14,12 +18,37 @@ const queryClient = new QueryClient({
 });
 
 const App: React.FunctionComponent = () => {
+  const dispatch = useAppDispatch();
+  React.useEffect(() => {
+    dispatch(configureApp());
+  }, [dispatch]);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [_, forceUpdate] = React.useReducer((x) => x + 1, 0);
```

Comment:
Ah, I meant to link this doc: https://reactjs.org/docs/hooks-faq.html#is-there-something-like-forceupdate which explains this line. Since `App` is a functional component we can't use `this.forceUpdate()` like we do in DataGateway (https://github.com/ral-facilities/datagateway/blob/73cd741772675501ad1b0bfde1c6d7626270f50d/packages/datagateway-dataview/src/App.tsx#L145) so this is the React recommended equivalent